src/unistd/src/lib.rs

```diff
@@ -116,13 +134,33 @@ pub extern "C" fn execve(
     argv: *const *mut c_char,
     envp: *const *mut c_char,
 ) -> c_int {
-    unimplemented!();
+    platform::execve(path, argv, envp)
 }
 
-#[no_mangle]
-pub extern "C" fn execvp(file: *const c_char, argv: *const *mut c_char) -> c_int {
-    unimplemented!();
-}
+/*
+ *#[no_mangle]
+ *pub extern "C" fn execvp(file: *const c_char, argv: *const *mut c_char) -> c_int {
```

Comment:
Just remove this if you're going to comment it out.

Reply:
I was hoping someone could have a look at it and see a way to fix it.